src/modelValidator.js

```diff
@@ -354,4 +409,15 @@ class ModelValidator {
   }
 }
 
+class ValidationMessage {
```

Comment:
I like the addition of this class a lot. I'd like to unify this with the 'ValidationError' JsDoc type that exists. 

Could you:
- Move the jsDoc for ValidationError to this class
- Unify the names. I have a slight preference for ValidationError over Validation method as its more descriptive, but I would be okay with ValidationMessage as well.
- Refactor the other ValidationError functions to use this constructor instead of constructing the object inline
- Possibly move this class to its own file. (I'm not a 100% on this, but I think it makes sense given the length of this file and I don't think it would break anything)